webpack/redux/reducers/RedHatRepositories/enabled.js

```diff
@@ -58,39 +59,45 @@ const changeRepoState = (state, repoToChange, stateDiff) => (
 );
 
 export default (state = initialState, action) => {
-  if (action.type === DISABLE_REPOSITORY_REQUEST) {
-    return changeRepoState(state, action.repository, { loading: true });
-  } else if (action.type === DISABLE_REPOSITORY_FAILURE) {
-    return changeRepoState(state, action.payload.repository, { loading: false });
-  } else if (action.type === ENABLED_REPOSITORIES_REQUEST) {
-    if (!action.silent) {
-      return state.set(['loading'], true);
-    }
-  } else if (action.type === ENABLED_REPOSITORIES_SUCCESS) {
-    const {
-      page, per_page, subtotal, results, // eslint-disable-line camelcase
-    } = action.response;
-
-    return Immutable({
-      repositories: mapRepositories(results),
-      pagination: {
-        page: Number(page) || 1,
-        // server can return per_page: null when there's error in the search query,
-        // don't store it in such case
-        // eslint-disable-next-line camelcase
-        perPage: Number(per_page || state.pagination.perPage),
-      },
-      itemCount: Number(subtotal),
-      loading: false,
-      searchIsActive: !isEmpty(action.search),
-      search: action.search,
-    });
-  } else if (action.type === ENABLED_REPOSITORIES_FAILURE) {
-    return Immutable({
-      repositories: [],
-      loading: false,
-      error: action.error,
-    });
+  switch (action.type) {
+    case DISABLE_REPOSITORY_REQUEST:
+      return changeRepoState(state, action.repository, { loading: true });
+    case DISABLE_REPOSITORY_FAILURE:
+      return changeRepoState(state, action.payload.repository, { loading: false });
+    case ENABLED_REPOSITORIES_REQUEST:
+      if (!action.silent) {
+        return state.set(['loading'], true);
+      }
+      return state;
+    case ENABLED_REPOSITORIES_SUCCESS: // eslint-disable-line no-case-declarations
+      const {
```

Comment:
Does it help to wrap the case blocks in brackets. As per documentation for [eslint-disable-line no-case-declarations](https://eslint.org/docs/rules/no-case-declarations), that should be accepted.

Reply:
@sjha4 cool, I'm sold.  I'll add the block.  thanks!

Reply:
@waldenraines  ^^